astroid/raw_building.py

```diff
@@ -402,7 +389,13 @@ def imported_member(self, node, member, name):
                 attach_dummy_node(node, name, member)
                 return True
 
-        real_name = {"gtk": "gtk_gtk", "_io": "io"}.get(modname, modname)
+        # On PyPy during bootstrapping we infer _io while _module is
+        # builtins. In CPython _io names itself io, see http://bugs.python.org/issue18602
+        # Therefore, this basically checks whether we are not in PyPy.
+        if modname == "_io" and not self._module.__name__ == "builtins":
+            return False
+
+        real_name = {"gtk": "gtk_gtk"}.get(modname, modname)
```

Comment:
I don't understand what this does. Is it it equivalent to this ?
```suggestion
        real_name = "gtk_gtk" if modname == "gtk" else modname
```

Reply:
Yes, both work. There doesn't actually seem to be a test for this so I just tried to keep the code as similar to the previous iteration as possible.

Reply:
```suggestion
        MODNAME_ALIASES = {"gtk": "gtk_gtk"} # Actually defined outside of the function scope ?
        real_name = MODNAME_ALIASES .get(modname, modname)
```
Would help with comprehension I guess.

Reply:
We could then also use your earlier suggestion? If we're going to change stuff anyway. Let me know what has you preference (or just commit directly)

Reply:
It depends on how likely we are to have another modname aliases, it's probably a good thing if we can extend to two aliases again without too much refactor. Let's keep the current code.